Makefile

```diff
@@ -51,7 +51,7 @@ deploy: manifests
 
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: controller-gen
-	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=ks-devops webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+	$(CONTROLLER_GEN) $(CRD_OPTIONS) rbac:roleName=ks-devops webhook paths="./pkg/api/..." output:crd:artifacts:config=config/crd/bases
```

Comment:
The RBAC annotations exist in both API and controller. You could exclude the `pkg/external`.

Reply:
You're right. But I have no idea to how exclude the `pkg/external` path in this situation.